src/main/groovy/edu/oregonstate/mist/classsearchapi/dao/ClassSearchDAO.groovy

```diff
@@ -191,7 +191,7 @@ class ClassSearchDAO {
             query['keyword'] = q.trim()
         }
         if (pageNumber && pageNumber > 1) {
-            query['offset'] = pageSize * pageNumber
+            query['offset'] = pageSize * (pageNumber -1)
```

Comment:
This is super nitpicky, but the whitespace around the subtraction operator is unbalanced.